apps/prairielearn/src/sprocs/check_assessment_access.sql

```diff
@@ -25,7 +25,59 @@ DECLARE
     active_access_rule_id bigint;
     next_active_start_date TIMESTAMP WITH TIME ZONE;
     next_active_credit integer;
+    override_group_id bigint;
+    end_date_from_override TIMESTAMP WITH TIME ZONE;
+    credit_from_override integer;
+    start_date_from_override TIMESTAMP WITH TIME ZONE;
 BEGIN
+    SELECT g.id  
+    INTO override_group_id
+    FROM groups as g JOIN group_configs AS gc 
+                ON g.group_config_id = gc.id 
+        JOIN group_users AS gu 
+                ON gu.group_id = g.id 
+    WHERE gc.assessment_id = check_assessment_access.assessment_id 
+        AND gu.user_id = check_assessment_access.user_id 
+        AND g.deleted_at IS NULL;
```

Comment:
Sorry we don't have automatic formatting for sprocs!

```suggestion
    SELECT
        g.id
    INTO
        override_group_id
    FROM
        groups as g
        JOIN group_configs AS gc ON (g.group_config_id = gc.id)
        JOIN group_users AS gu ON (gu.group_id = g.id)
    WHERE
        gc.assessment_id = check_assessment_access.assessment_id
        AND gu.user_id = check_assessment_access.user_id
        AND g.deleted_at IS NULL;
```

Reply:
Changed in ee2d44c.